### DISCUSSION TITLE
close dropdown on select in multi select view

### DISCUSSION THREAD

**Discussion by rajeevprasanna** [dowjones/react-dropdown-tree-select#503] [Ideas]
Right now, the dropdown is closing on selection in a single select view. But we have a need to close the dropdown on the multi-select view also. The reason for that is we want to trigger some action immediately on selection and refresh the view.

is it possible to add this feature?  i can raise pull request if it is allowed.

**Comment by mrchief** [comment]
> But we have a need to close the dropdown on the multi-select view also

How do you know when the user has finished selecting? If you do not want multiple selects, then you're probably looking for https://github.com/dowjones/react-dropdown-tree-select#radioselect

**Reply by rajeevprasanna** [reply]
user will open the dropdown again to select additional fields. let the default behavior of keeping dropdown open. but if some feature wants to close the view, why block it? give that flexibility to developers to choose behavior.

**Reply by mrchief** [reply]
So you'd open the dropdown, choose a value, the dropdown will close and then the user will have to open the dropdown again to choose the 2nd value, the dropdown will close and this cycle repeats? That's a terrible UX for multi-select!
